migrate/20240209_postgres_multiple_servers.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  no_transaction
```

Comment:
huh, why? I see you also are doing "concurrently" but is that necessary? You aren't at a big company anymore, for now :smile:

Reply:
rubocop complains if we don't do concurrently with index creations. I could add override, but this seemed OK as well.

Reply:
huh weird. I guess this is @cyberdelia being very keyed into the Heroku production case in. Alright, it survives today, more trouble than losing the tooling.